Fix globals, part 6: `performance` (#33339)
mdn/content: files/en-us/web/api/workerglobalscope/index.md

  - : Returns the {{domxref("WorkerNavigator")}} associated with the worker. It is a specific navigator object, mostly a subset of the {{domxref("Navigator")}} for browsing scopes, but adapted to workers.
- {{domxref("WorkerGlobalScope.origin")}} {{ReadOnlyInline}}
  - : Returns the global object's origin, serialized as a string.
- {{domxref("WorkerGlobalScope.performance")}} {{ReadOnlyInline}}

>   - : Returns the {{domxref("Performance")}} associated with the worker. It is a regular performance object, except that only a subset of its property and methods are available to workers.

Reviewer: ```suggestion
  - : Returns the {{domxref("Performance")}} associated with the worker. Only a subset of the  properties and methods of the `Performance` interface are available to workers.
```
